src/outbox/outboxActions.js

```diff
@@ -67,7 +67,10 @@ export const trySendMessages = (dispatch: Dispatch, getState: GetState): boolean
       // prettier-ignore
       const to =
         item.type === 'private'
-          ? item.narrow[0].operand
+            // This will include the self user, possibly twice.  That's
+            // fine; on send, the server (since at least 2013) drops dupes
+            // and normalizes whether to include the sender.
+          ? item.display_recipient.map(r => r.email).join(',')
```

Comment:
I wouldn't want to just `.slice` off an element – but it does feel like the self-user should be `.filter`ed out here, rather than relying on the permissiveness of the server implementation.

Also, if these are emails rather than IDs, we should probably use JSON rather than CSV. (If I'd realized that's what they were, I would have done the additional testing on #3734. :disappointed:) I'm not about to ask you to do that for this PR, but tagging it with an inline `TODO:` would be appreciated.

Reply:
In #4299 I ended up taking care of that duplicate self-user by fixing how we construct `display_recipient` in the first place, elsewhere in this file. Having it wrong there turns out to cause its own buggy behavior, before any request even reaches the server.